builtin/mainmenu/tab_content.lua

```diff
@@ -142,7 +142,7 @@ local function get_formspec(tabview, name, tabdata)
 			end
 		end
 
-		table.insert_all(retval, {"textarea[7.1,2.4;8,3.1;;;", desc, "]"})
+		table.insert_all(retval, {"textarea[7.1,2.4;8,3.1;;;", core.formspec_escape(desc), "]"})
```

Comment:
This will double escape the "fgettext". I believe you either need to use `fgettext_ne` or only escape `desc = info.description` above

Reply:
I see. Escaping info.description sounds like a better idea, then.